microservices/embeddingservice/src/main.py

```diff
@@ -43,7 +43,16 @@ def add_embedding_to_document(data: AddEmbeddingToDocRequest):
 
 @router.get("/models")
 def get_models():
-    return list(text_embedder.models.keys())
+    # model_config = config.get("models", default="")
+    return [
+        config["models"]["c2c_models"][model]["model_name"]
```

Comment:
could be simplified by iterating over `config["models"]["c2c_models"].items()`

Reply:
Done